crates/ruff_linter/src/rules/flake8_annotations/helpers.rs

```diff
@@ -143,22 +143,27 @@ impl AutoPythonType {
                 Some((expr, vec![no_return_edit]))
             }
             AutoPythonType::Atom(python_type) => {
-                let expr = type_expr(python_type)?;
-                Some((expr, vec![]))
+                let (expr, edits) = type_expr(python_type, checker, at)?;
+                Some((expr, edits))
```

Comment:
nit: I think this is the same


```suggestion
                type_expr(python_type, checker, at)
```